docs/fairness-indicators.md

```diff
@@ -0,0 +1,58 @@
+# Evaluating Models with the Fairness Indicators Dashboard [Beta]
+
+![What-If Tool](./images/fairness-indicators.png)
```

Comment:
the alt text for this image should not be `What-If Tool`

Reply:
Done